phases/08-generative-ai/14-evaluation-fid-clip-score/code/main.py

```
@@ -0,0 +1,146 @@
+        pivot = col
+        for r in range(col + 1, n):
+            if abs(A[r][col]) > abs(A[pivot][col]):
+                pivot = r
+        A[col], A[pivot] = A[pivot], A[col]
+        piv = A[col][col]
+        if abs(piv) < 1e-12:
+            piv = 1e-12
+        for j in range(2 * n):
+            A[col][j] /= piv
+        for r in range(n):
+            if r == col: continue
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)